### PR TITLE
Ignore xmlns without CLR assembly name on type resolution (`using` syntax)

### PR DESCRIPTION
**Pull request by YohDeadfall**
## What does the pull request do?

Fixes an error in Avalonia which hides the original error which must be thrown instead.

## What is the current behavior?

Feed an empty string to `AssemblyName` and fail with a stack trace pointing to it and internals.

## What is the updated/expected behavior with this PR?

Tell the user that the requested type cannot be resolved.

## How was the solution implemented (if it's not obvious)?

By checking the length of a CLR assembly name.

## Checklist

- [ ] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [x] Consider submitting a PR to https://github.com/AvaloniaUI/Documentation with user documentation

## Breaking changes

No.

## Obsoletions / Deprecations

No.

## Fixed issues

Decided not to open it.

### CODE REVIEW

**Review by maxkatz6 — APPROVED** [review]
Honestly, it feels like a solvable bug (aside from trimming problems which are unavoidable).

But this PR still makes it easier to diagnose, so LGTM.

File: src/Markup/Avalonia.Markup.Xaml/XamlIl/Runtime/XamlIlRuntimeHelpers.cs [hunk 1]
```diff
@@ -154,7 +154,8 @@ public Type Resolve(string qualifiedTypeName)
                 var namespaces = _nsInfo.XmlNamespaces;
                 if (!namespaces.TryGetValue(ns, out var lst))
                     throw new ArgumentException("Unable to resolve namespace for type " + qualifiedTypeName);
-                foreach (var entry in lst)
+                var resolvable = lst.Where(static e => e.ClrAsseblyName is { Length: > 0 });
```
Comment (maxkatz6) [inline src/Markup/Avalonia.Markup.Xaml/XamlIl/Runtime/XamlIlRuntimeHelpers.cs:157]: @YohDeadfall there is a compile error here

**Comment by YohDeadfall** [comment]
It can be solved by scanning all loaded assemblies which is bad for performance. Another solution is improving XamlX which can add all assemblies having the specified namespace, but I have no idea how to do that even knowing the exact lines which generate an interface implementation.

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0034938-beta`. (feed url: https://pkgs.dev.azure.com/AvaloniaUI/AvaloniaUI/_packaging/avalonia-all/nuget/v3/index.json) [PRBUILDID]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0035013-beta`. (feed url: https://pkgs.dev.azure.com/AvaloniaUI/AvaloniaUI/_packaging/avalonia-all/nuget/v3/index.json) [PRBUILDID]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0035036-beta`. (feed url: https://pkgs.dev.azure.com/AvaloniaUI/AvaloniaUI/_packaging/avalonia-all/nuget/v3/index.json) [PRBUILDID]

**Pull request merged**